lms/djangoapps/courseware/views/views.py

```diff
@@ -161,6 +161,15 @@
     cert_web_view_url=None
 )
 
+COURSE_HOMEPAGE_INVERT_TITLE =\
+    configuration_helpers.get_value('COURSE_HOMEPAGE_INVERT_TITLE', False)
+
+COURSE_HOMEPAGE_SHOW_SUBTITLE =\
+    configuration_helpers.get_value('COURSE_HOMEPAGE_SHOW_SUBTITLE', True)
+
+COURSE_HOMEPAGE_SHOW_ORG =\
+    configuration_helpers.get_value('COURSE_HOMEPAGE_SHOW_ORG', True)
```

Comment:
These values need to be set in the actual view before they're used, because they need a current active request in order to query the current site configuration.